karma.conf.js

```diff
@@ -42,6 +39,13 @@ export default function configKarma(karma : Object) {
         ...karmaConfig.files
     ];
 
+    karmaConfig.customLaunchers = {
+        Chrome_travis_ci: {
```

Comment:
Was one of the flags passed to chrome in grumbler-scripts causing the problem?

Reply:
No, I think the problem was the version of Chrome installed by Aptitude. This launcher config just ensures we use Chrome headless for Travis for consistency sake.

Reply:
ok gotcha. then I'd say let's stick to grumbler-script's chrome definition if it's not specifically causing any problems. then we get the benefit of any flags we want to pass through the command line (like `--dev-tools`)